content/docs/develop/operate/upgrade/index.md

```diff
@@ -7,17 +7,69 @@ keywords: upgrade
 
 ## Upgrade from 0.6.2 to 0.7
 
-* To download and install the Keptn CLI for version 0.7.0, please refer to the [Install Keptn CLI section](../setup-keptn/#install-keptn-cli).
+1. To download and install the Keptn CLI for version 0.7.0, please refer to the [Install Keptn CLI section](../setup-keptn/#install-keptn-cli).
 
-* To upgrade your Keptn installation from 0.6.2 to 0.7.0, you can deploy a *Kubernetes Job* that will take care of updating all components to the 0.7.0 release. Please [verify that you are connected to the correct Kubernetes cluster](../../reference/troubleshooting/#verify-kubernetes-context-with-keptn-installation)
-before deploying the upgrading job with the next command:
+1. To upgrade your Keptn installation from 0.6.2 to 0.7.0, a *Kubernetes Job* is provided that upgrades all components to the 0.7.0 release. 
+
+    * Please [verify that you are connected to the correct Kubernetes cluster](../../reference/troubleshooting/#verify-kubernetes-context-with-keptn-installation)
+before deploying the upgrading job.
+
+    * Keptn 0.7 uses Helm 3 while previous Keptn releases rely on Helm 2. (Please take into account that the end-of-life period of Helm 2 begins on [August 13th, 2020](https://helm.sh/blog/covid-19-extending-helm-v2-bug-fixes/).)
+
```

Comment:
Let's tell the user the impact of upgrading from Helm 2 to Helm 3.

For instance, it is necessary to upgrade all helm releases, old releases won't work anymore.

Reply:
I made it clear what it means when not upgrading.